src/metrics/prometheus.go

```diff
@@ -24,6 +26,23 @@ func Push(config *core.Configuration) {
 		return
 	}
 
+	if config.Metrics.PushHostinfo {
+		name, _ := os.Hostname()
+		gauge := prometheus.NewGauge(prometheus.GaugeOpts{
+			Namespace: "plz",
+			Subsystem: "metrics",
+			Name:      "hostinfo",
+			Help:      "Please host running info",
+			ConstLabels: prometheus.Labels{
+				"remote":   strconv.FormatBool(config.Remote.URL != ""),
```

Comment:
prefer `config.NumRemoteExecutors() > 0`, that handles both the URL and whether we're configured to execute anything

maybe we should add an explicit function for this too, there are several cases where things ask this question